Sources/ContentIntent.swift

```diff
@@ -108,21 +109,82 @@ public extension Android.Content.Intent {
     }
     
     /**
-     * Add extended data to the intent. (byte)
+     * Retrieve extended data from the intent. (Byte array)
      */
-    public func putExtra(name: String, value: Int8) -> Android.Content.Intent {
+    public func getByteArrayExtra(name: String) -> [Int8]? {
+        var __locals = [jobject]()
+        
+        var __args: [jvalue] = [
+            JNIType.toJava(value: name, locals: &__locals)
+        ]
         
+        let __return = JNIMethod.CallObjectMethod(object: javaObject,
+                                                   methodName: "getByteArrayExtra",
+                                                   methodSig: "(Ljava/lang/String;)[B",
+                                                   methodCache: &JNICache.MethodID.getByteArrayExtra,
+                                                   args: &__args,
+                                                   locals: &__locals)
+        
+        return JNIType.toSwift(type: [Int8].self, from: __return)
+    }
+    
+    /**
+     * Retrieve extended data from the intent. (Byte)
+     */
+    public func getByteExtra(name: String) -> Int8 {
         var __locals = [jobject]()
+        
+        var __args: [jvalue] = [
+            JNIType.toJava(value: name, locals: &__locals)
+        ]
+        
+        let __return = JNIMethod.CallBooleanMethod(object: javaObject,
```

Comment:
you should call the method `JNIMethod.CallByteMethod()` because the method `getByteExtra()` returns an Int8(byte)